### DISCUSSION TITLE
Request

### DISCUSSION THREAD

**Discussion by Princeofmaliciouscode** [monkeytypegame/monkeytype#1051] [Ideas] [answered]
Can there be a desktop version too?

**Comment by Miodec** [comment, marked as answer]
Answered in #1018 - i don't really have any experience in making desktop apps and its not a priority at the moment.

**Comment by unrealapex** [comment]
The closest thing to what you mentioned is a shortcut on google chrome. If you go the the menu, click `more tools`, click `create a shortcut`, and check the -[x] open as window option, monkeytype will open up as a fullscreen pwa. If you are referring to an actual app, do you mean an Electron app like Visual Studio Code, Discord, or Twitch ?
